src/libs/actions/Report.js

```diff
@@ -158,10 +158,9 @@ function getSimplifiedReportObject(report) {
     if (report.reportActionCount > 0) {
         // We are removing any html tags from the message html since we cannot access the text version of any comments as
         // the report only has the raw reportActionList and not the processed version returned by Report_GetHistory
-        // We convert the line-breaks in html to space ' ' before striping the tags
-        lastMessageText = lastActionMessage
-            .replace(/((<br[^>]*>)+)/gi, ' ')
-            .replace(/(<([^>]+)>)/gi, '');
+        // We convert the line-breaks and blockquote closing tag in html to space ' ' before striping the tags
```

Comment:
I don't think this comment makes sense here. This is best suited for `htmlToText` method.